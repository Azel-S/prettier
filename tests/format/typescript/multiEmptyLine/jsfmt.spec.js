run_spec(__dirname, ["babel", "flow", "typescript"], {multiEmptyLine: false});
run_spec(__dirname, ["babel", "flow", "typescript"], {multiEmptyLine: true});