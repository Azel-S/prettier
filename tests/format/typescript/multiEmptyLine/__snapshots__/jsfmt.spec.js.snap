// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`multiEmptyLine.ts - {"multiEmptyLine":false} format 1`] = `
====================================options=====================================
multiEmptyLine: false
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
function add(x, y) {


    //comment
    

    return x + y;

    //comment



  }

  function addTwo(x, y) {


    //comment
    

    return x + y;

   



  }

  function addThree(x, y) {


    
    

    return x + y;

    //comment



  }

  function addFour(x, y) {


    
    

    return x + y;

 



  }
=====================================output=====================================
function add(x, y) {
  //comment

  return x + y;

  //comment
}

function addTwo(x, y) {
  //comment

  return x + y;
}

function addThree(x, y) {
  return x + y;

  //comment
}

function addFour(x, y) {
  return x + y;
}

================================================================================
`;

exports[`multiEmptyLine.ts - {"multiEmptyLine":true} format 1`] = `
====================================options=====================================
multiEmptyLine: true
parsers: ["babel", "flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
function add(x, y) {


    //comment
    

    return x + y;

    //comment



  }

  function addTwo(x, y) {


    //comment
    

    return x + y;

   



  }

  function addThree(x, y) {


    
    

    return x + y;

    //comment



  }

  function addFour(x, y) {


    
    

    return x + y;

 



  }
=====================================output=====================================
function add(x, y) {


  //comment

  return x + y;

  //comment



}

function addTwo(x, y) {


  //comment

  return x + y;





}

function addThree(x, y) {





  return x + y;

  //comment



}

function addFour(x, y) {





  return x + y;





}

================================================================================
`;
